cfgov/filing_instruction_guide/jinja2/filing_instruction_guide/section.html

```diff
@@ -1,50 +1,35 @@
-<div class="block block__flush-bottom">
-  {% set block_type = block.block_type %}
-  {% set block_level = {
-    'Fig_Section': 'h2',
-    'Fig_Subsection': 'h3'
-    }[block_type] | default('h4')
-  %}
-  {% if block_type in ['Fig_Section', 'Fig_Subsection'] %}
-      {% set header = value.section_id + " " + value.header %}
-  {% else %}
-      {% set header = value.header %}
-  {% endif %}
+{% set block_type = block.block_type %}
+{% set block_types = {
+    'Fig_Section': {
+        'heading': 'h2',
+        'class': 'o-fig_section'
+    },
+    'Fig_Subsection': {
+        'heading': 'h3',
+        'class': 'o-fig-section__sub'
+    },
+    'Fig_Level_3_Subsection': {
+        'heading': 'h4',
+        'class': 'o-fig-section__sub-sub'
+    },
+} %}
+{% set block_heading_level = block_types[block_type].heading | default('h4') %}
+{% set block_class = block_types[block_type].class | default('o-fig_section') %}
 
-  <div class="block block__flush-bottom">
-    <{{block_level}} class="report-header">
-            <a id="{{ value.section_id }}"
-                href="#{{ value.section_id }}">
-                    {{header}}
-            </a>
-    </{{block_level}}>
-  </div>
-{% if value.content %}
-    <div class="o-full-width-text-group">
-        {% for block in value.content %}
-            {% if 'content' in block_type %}
-                <div class="m-full-width-text">
-                    {{ block.value | safe }}
-                </div>
-            {% elif 'table' in block_type %}
```

Comment:
Unfortunately we do need to keep this `if` case for when the block is a table. If we don't have the `autoescape off` commands, it'll render like this:

![Screen Shot 2022-07-01 at 1 58 38 PM](https://user-images.githubusercontent.com/4295388/176947026-d8ed78b1-0560-49b3-9d1a-67e14b20e12d.png)

I tried a well and it looked fine, so the rest of these if-else statements might not be necessary.

Reply:
Also, is it a regression that the header links are now blue? It might have been the `report-header` class that was making them black before.

Reply:
Ah yes, let me add more content locally to test all the block types. My bad.

Reply:
@schbetsy I tried to unwind all the conditional logic only to realize that none of the conditions are ever met because `block_type` is always [one of these](https://github.com/cfpb/consumerfinance.gov/blob/6c73741800bbdbaf6830cf5691fbe5a91ae4395c/cfgov/filing_instruction_guide/models.py#L41-L43). So all this logic can be reduced to `{{ block | safe }}`.

And the heading colors have been reset to black. Good eye. When I do more TOC stuff I'll be renaming CSS classes.